ocaml-lsp-server/test/e2e-new/code_actions.mli

```diff
@@ -18,6 +18,13 @@ val apply_code_action
   -> Range.t
   -> string option
 
+val apply_code_action_no_edit
+  :  ?diagnostics:Diagnostic.t list
+  -> string
+  -> string
+  -> Range.t
+  -> unit option
```

Comment:
The return type is rather weird. What about a variant to explain what you're returning

Reply:
Actually I don't even see where this function is being used

Reply:
I will remove it. It was "relevant" in an earlier version of this implementation but no longer needed now.